test/shutdown.test.ts

```diff
@@ -35,7 +35,7 @@ const SECONDS = ONE_SECOND;
 const GRACE_PERIOD = 1000;
 
 describe("cli shutdown routine", function () {
-  this.timeout(30 * SECONDS);
+  this.timeout(60 * SECONDS);
```

Comment:
@joshwlewis this specific test fails 50% of the time in GHA in their mac runners which is a pain.  and the test doesn't always take the full minute but it will wait that long before triggering the timeout.  this is annoying but seems slightly less annoying that having to constantly re-trigger the failed GHA steps.